support-frontend/assets/components/svgs/threeTierHeadlineImageDesktop.tsx

```diff
@@ -0,0 +1,160 @@
+				fill="white"
+			/>
+			<path
+				d="M27.8467 19.0797L25.1797 17.924V15.9683L34.9142 14.6348L35.981 14.7237V32.6814C35.981 33.837 36.2477 34.6371 36.7811 35.0816C37.3145 35.4965 38.0405 35.7039 38.9591 35.7039C39.5518 35.7039 40.1 35.6447 40.6037 35.5261C41.1371 35.378 41.6409 35.2002 42.115 34.9927V19.0797L39.448 17.924V15.9683L49.1825 14.6348L50.1604 14.7237V35.7039L52.783 37.0374V38.8599L43.1818 39.9711L42.2039 39.8822V36.993H41.9372C41.0482 37.8227 39.9814 38.5487 38.7369 39.171C37.5219 39.7637 36.1588 40.06 34.6475 40.06C32.3361 40.06 30.6174 39.4229 29.4913 38.1487C28.3949 36.8744 27.8467 35.2002 27.8467 33.1258V19.0797Z"
+				fill="white"
+			/>
+			<path
+				d="M11.0457 41.4661C9.23808 41.4661 7.44527 41.318 5.66728 41.0216C3.8893 40.7549 2.42245 40.3697 1.26676 39.8659L0.822266 31.7761H4.64494L6.37848 37.6879C6.94151 37.9546 7.57862 38.162 8.28982 38.3102C9.03065 38.4287 9.81592 38.488 10.6457 38.488C12.5125 38.488 13.8905 38.1176 14.7795 37.3768C15.6685 36.6359 16.113 35.584 16.113 34.2208C16.113 32.9466 15.6537 31.9539 14.735 31.2427C13.8164 30.5019 12.3792 29.7314 10.4234 28.9313L8.68986 28.2201C6.20068 27.183 4.30416 25.9087 3.0003 24.3974C1.72608 22.8565 1.08896 20.8118 1.08896 18.2634C1.08896 16.8706 1.31121 15.5668 1.75571 14.3518C2.20021 13.1072 2.89658 12.0404 3.84485 11.1514C4.79311 10.2624 6.00807 9.56605 7.48972 9.06229C9.00101 8.55852 10.7938 8.30664 12.8681 8.30664C15.061 8.30664 16.839 8.45481 18.2021 8.75114C19.5652 9.01784 20.6617 9.29935 21.4914 9.59568L21.9359 17.3744H18.291L16.6908 11.7737C16.2759 11.5959 15.7426 11.4626 15.0906 11.3737C14.4387 11.2551 13.6682 11.1959 12.7792 11.1959C11.2679 11.1959 10.1271 11.5663 9.35661 12.3071C8.58615 13.0183 8.20092 13.9814 8.20092 15.1963C8.20092 16.2928 8.57133 17.1818 9.31216 17.8633C10.0826 18.5449 11.342 19.2116 13.0904 19.8636L14.8684 20.5748C16.2611 21.1082 17.5057 21.6564 18.6021 22.2194C19.7282 22.7528 20.6617 23.4047 21.4025 24.1752C22.1729 24.916 22.7508 25.8198 23.136 26.8866C23.5509 27.9534 23.7583 29.2573 23.7583 30.7982C23.7583 34.206 22.6323 36.8434 20.3801 38.7103C18.128 40.5475 15.0165 41.4661 11.0457 41.4661Z"
+				fill="white"
+			/>
+		</svg>
```

Comment:
```
<svg width="896" height="50" viewBox="0 0 896 50" xmlns="http://www.w3.org/2000/svg" aria-labelledby="dancingTitleID">
<title>Support fearless, independent journalism</title>
 <path d="m855 37 2.5-1.1v-17l-2.7-1.1v-1.9l9.6-1.4 0.98 0.089v2.8h0.27c1-0.92 2.2-1.7 3.4-2.2 1.2-0.56 2.6-0.84 4.2-0.84 1.4 0 2.5 0.24 3.4 0.71s1.6 1.3 2 2.4h0.18c1.1-0.95 2.3-1.7 3.5-2.3 1.3-0.56 2.7-0.84 4.3-0.84 2 0 3.5 0.53 4.6 1.6 1.1 1 1.6 2.7 1.6 4.9v15l2.6 1.1v1.9h-13v-1.9l2-1.1v-14c0-1.2-0.25-2.1-0.76-2.6-0.5-0.53-1.2-0.8-2.1-0.8-0.5 0-1 0.074-1.5 0.22-0.5 0.15-0.96 0.31-1.4 0.49v17l2 1.1v1.9h-12v-1.9l2-1.1v-14c0-1.2-0.24-2.1-0.71-2.6s-1.2-0.8-2.1-0.8c-0.56 0-1.1 0.074-1.6 0.22-0.47 0.15-0.92 0.33-1.3 0.53v17l2 1.1v1.9h-13v-1.9z" fill="#fff"/>
 <path d="m846 23c2.5 0.62 4.4 1.6 5.6 2.8 1.2 1.2 1.7 3 1.7 5.3 0 2.7-0.84 4.8-2.5 6.3-1.7 1.5-4.3 2.2-8 2.2-1.4 0-2.9-0.12-4.4-0.36-1.5-0.21-2.8-0.53-4-0.98l-0.27-6.4h2.9l1.9 4.2c0.47 0.27 1 0.47 1.7 0.62s1.3 0.22 2.1 0.22c1.4 0 2.4-0.28 3-0.84s0.98-1.3 0.98-2.2c0-0.8-0.25-1.4-0.76-1.9-0.48-0.47-1.3-0.86-2.6-1.2l-1.6-0.4c-2.6-0.65-4.5-1.6-5.7-2.9-1.2-1.3-1.9-3.1-1.9-5.4 0-2.4 0.82-4.3 2.4-5.8 1.7-1.5 4.2-2.3 7.6-2.3 1.4 0 2.8 0.089 4.1 0.27 1.3 0.15 2.5 0.4 3.4 0.76v6h-2.9l-1.3-3.8c-0.36-0.18-0.8-0.31-1.3-0.4-0.5-0.089-1.1-0.13-1.7-0.13-1.2 0-2.1 0.25-2.7 0.76-0.59 0.5-0.89 1.2-0.89 2.1 0 0.8 0.25 1.4 0.76 1.9 0.5 0.44 1.4 0.84 2.8 1.2l1.6 0.4z" fill="#fff"/>
 <path d="m826 4.1c1.2 0.024 2.2 0.44 3 1.3 0.84 0.82 1.3 1.8 1.2 3-0.025 1.2-0.48 2.2-1.4 3-0.88 0.75-1.9 1.1-3 1.1-1.2-0.024-2.2-0.43-3-1.2-0.81-0.82-1.2-1.8-1.2-3 0.024-1.2 0.46-2.2 1.3-3s1.9-1.2 3-1.1zm-6.8 11 10-1.2 0.84 0.11-0.45 22 2.5 1.2-0.04 1.9-13-0.27 0.039-1.9 2.6-1.1 0.36-17-2.7-1.1 0.039-1.9z" fill="#fff"/>
 <path d="m805 4.8 9.9-1.2 0.98 0.15-0.66 32 2.5 1.2-0.04 2-13-0.27 0.041-2 2.6-1.1 0.57-28-2.7-1.1 0.041-2z" fill="#fff"/>
 <path d="m788 25 5.3-1.3-0.15-3.3c-0.074-1.7-0.42-2.8-1-3.6-0.62-0.71-1.6-1-3.1-0.98-0.42 0.018-0.8 0.065-1.2 0.14-0.32 0.044-0.65 0.12-0.97 0.22l-1.4 5.4-3.6 0.16-0.3-6.7c1.1-0.41 2.4-0.81 3.9-1.2 1.5-0.42 3.1-0.67 5-0.76 3.3-0.15 5.8 0.27 7.7 1.3 1.9 0.98 2.9 2.9 3 5.6l0.64 14 2.4 1.1 0.072 1.6c-0.4 0.31-1 0.59-1.9 0.84-0.82 0.27-1.7 0.43-2.7 0.48-1.5 0.068-2.7-0.13-3.5-0.6-0.82-0.5-1.4-1.2-1.8-2l-0.27 0.012c-0.67 0.95-1.5 1.7-2.5 2.3-1 0.55-2.3 0.86-3.8 0.93-2 0.091-3.7-0.41-5.1-1.5-1.3-1.1-2-2.7-2.1-4.8-0.091-2 0.47-3.7 1.7-4.9 1.2-1.2 3.1-2.1 5.7-2.7zm3.6 9.6c0.47-0.021 0.88-0.099 1.2-0.23 0.38-0.17 0.67-0.36 0.86-0.57l-0.34-7.5-1.9 0.31c-1.1 0.17-1.9 0.59-2.4 1.3-0.47 0.67-0.68 1.7-0.62 3.1 0.069 1.5 0.4 2.6 0.98 3.1 0.58 0.48 1.3 0.7 2.2 0.66z" fill="#fff"/>
 <path d="m750 34 2.6-0.9 1.7-17-2.6-1.3 0.18-1.9 9.8-0.47 0.96 0.18-0.28 2.8 0.27 0.026c1.1-0.81 2.4-1.4 4-1.8 1.5-0.42 3-0.55 4.6-0.4 2.1 0.21 3.6 0.87 4.4 2s1.2 2.8 0.98 5l-1.5 15 2.4 1.4-0.19 1.9-13-1.3 0.19-1.9 2.4-0.8 1.4-14c0.12-1.2-0.061-2.1-0.54-2.7-0.45-0.58-1.2-0.92-2.2-1-0.65-0.064-1.3-0.05-1.8 0.043-0.54 0.095-1.1 0.22-1.6 0.38l-1.6 17 2.1 1.2-0.19 1.9-13-1.3 0.19-1.9z" fill="#fff"/>
 <path d="m732 14 9.8-1.2 0.8 0.11-0.11 5.5 0.22 0.0046c0.69-2 1.6-3.4 2.6-4.3 1.1-0.96 2.3-1.4 3.7-1.4 0.21 0.0043 0.43 0.024 0.67 0.058 0.24 0.035 0.41 0.083 0.53 0.14l-0.15 7.3c-0.21-0.064-0.5-0.11-0.89-0.15-0.36-0.037-0.72-0.059-1.1-0.067-1.1-0.023-2.1 0.016-2.9 0.12-0.86 0.1-1.7 0.29-2.4 0.57l-0.29 14 3.3 1.2-0.039 1.9-14-0.29 0.04-1.9 2.6-1.1 0.36-17-2.6-1.1 0.039-1.9z" fill="#fff"/>
 <path d="m706 16-2.6-1.2 0.04-2 9.8-1.1 1.1 0.11-0.37 18c-0.024 1.2 0.23 2 0.75 2.4 0.52 0.43 1.2 0.65 2.2 0.67 0.59 0.012 1.1-0.036 1.6-0.14 0.54-0.14 1-0.3 1.5-0.5l0.33-16-2.6-1.2 0.041-2 9.8-1.1 0.98 0.11-0.43 21 2.6 1.4-0.037 1.8-9.6 0.91-0.98-0.11 0.06-2.9-0.27-0.0055c-0.91 0.81-2 1.5-3.2 2.1-1.2 0.57-2.6 0.84-4.1 0.8-2.3-0.048-4-0.72-5.1-2-1.1-1.3-1.6-3-1.5-5.1l0.29-14z" fill="#fff"/>
 <path d="m689 37c-1.8-0.038-3.5-0.37-5-0.99-1.5-0.65-2.8-1.5-3.9-2.7-1.1-1.1-1.9-2.5-2.5-4.1-0.56-1.6-0.82-3.4-0.78-5.4 0.04-2 0.37-3.7 1-5.3 0.62-1.6 1.5-2.9 2.6-3.9 1.1-1.1 2.4-1.9 4-2.5 1.6-0.56 3.3-0.82 5.1-0.78 1.9 0.039 3.6 0.37 5.1 0.99 1.5 0.62 2.8 1.5 3.9 2.6 1.1 1.1 1.9 2.4 2.4 4.1 0.59 1.6 0.86 3.4 0.82 5.3-0.04 2-0.37 3.7-1 5.3-0.62 1.6-1.5 2.9-2.6 4-1.1 1.1-2.5 1.9-4 2.5-1.6 0.56-3.3 0.82-5.1 0.78zm0.48-23c-0.59-0.012-1.1 0.12-1.6 0.41-0.45 0.29-0.85 0.81-1.2 1.6-0.34 0.76-0.62 1.8-0.82 3.1-0.18 1.3-0.28 3-0.32 5-0.043 2-4e-3 3.7 0.12 5.1 0.15 1.3 0.38 2.4 0.69 3.2 0.31 0.78 0.7 1.3 1.2 1.7 0.47 0.31 1 0.47 1.6 0.48 0.59 0.012 1.1-0.13 1.6-0.41 0.48-0.32 0.88-0.86 1.2-1.6 0.34-0.79 0.6-1.9 0.78-3.2 0.2-1.3 0.33-3 0.37-5.1 0.042-2-0.013-3.7-0.16-5-0.12-1.3-0.34-2.4-0.65-3.1-0.28-0.78-0.66-1.3-1.1-1.6-0.44-0.31-0.97-0.46-1.6-0.48z" fill="#fff"/>
 <path d="m670 0.9c1.2 0.024 2.1 0.44 3 1.3 0.81 0.82 1.2 1.8 1.2 3-0.025 1.2-0.46 2.2-1.3 3-0.84 0.75-1.8 1.1-3 1.1-1.2-0.024-2.2-0.43-3-1.2-0.81-0.82-1.2-1.8-1.2-3 0.024-1.2 0.46-2.2 1.3-3s1.9-1.2 3-1.1zm-4.3 14-2.7-1.2 0.04-2 10-1.3 0.84 0.11-0.53 26c-0.058 2.8-0.8 5-2.2 6.4-1.4 1.5-3.6 2.2-6.5 2.1-1-0.021-2-0.14-2.8-0.37-0.86-0.23-1.6-0.52-2.2-0.89l0.051-2.5c0.3 0.095 0.6 0.18 0.93 0.24 0.3 0.065 0.59 0.1 0.89 0.11 1.2 0.024 2.1-0.24 2.7-0.79 0.63-0.55 0.96-1.5 1-3l0.47-23z" fill="#fff"/>
 <path d="m632 15-3.4 0.41-0.25-2.1 3.3-1.6-0.51-4.3 7.9-2.2 0.66 5.6 5.3-0.63 0.39 3.3-5.3 0.63 1.7 14c0.15 1.2 0.47 2.1 0.98 2.6 0.53 0.47 1.5 0.63 2.8 0.48 0.38-0.045 0.78-0.11 1.2-0.19 0.44-0.082 0.79-0.15 1-0.21l0.27 2.3c-0.66 0.5-1.5 0.96-2.7 1.4-1.1 0.46-2.3 0.76-3.6 0.92-2.4 0.28-4.2-0.079-5.6-1.1-1.3-1-2.1-2.8-2.4-5.4l-1.7-14z" fill="#fff"/>
 <path d="m600 32 2.6-0.92 1.6-17-2.6-1.3 0.17-1.9 9.8-0.52 0.96 0.18-0.26 2.8 0.27 0.025c1.1-0.82 2.4-1.4 4-1.9 1.5-0.42 3-0.56 4.5-0.42 2.1 0.2 3.6 0.85 4.5 2 0.88 1.1 1.2 2.8 1 5l-1.4 15 2.4 1.4-0.18 1.9-13-1.2 0.18-1.9 2.4-0.81 1.3-14c0.11-1.2-0.072-2.1-0.55-2.7-0.45-0.58-1.2-0.91-2.2-1-0.65-0.06-1.3-0.043-1.8 0.052-0.54 0.098-1.1 0.23-1.6 0.39l-1.6 17 2.1 1.2-0.18 1.9-13-1.2 0.18-1.9z" fill="#fff"/>
 <path d="m599 24-14 0.36c0.18 2.5 0.83 4.3 2 5.5 1.1 1.2 3 1.7 5.7 1.7 1.3-0.031 2.4-0.18 3.4-0.44 0.97-0.29 1.9-0.67 2.7-1.1l0.055 2.2c-0.63 0.93-1.8 1.9-3.4 2.8-1.6 0.9-3.6 1.4-5.9 1.4-4.1 0.1-7.3-0.98-9.4-3.2-2.2-2.3-3.3-5.4-3.4-9.4-0.096-3.9 0.91-7 3-9.3 2.1-2.3 5.1-3.5 8.8-3.6 3.5-0.086 6.2 0.83 8 2.7s2.8 5.2 3 9.8l0.015 0.62zm-11-11c-1.1 0.028-2 0.72-2.7 2.1-0.62 1.3-0.89 3.5-0.82 6.5l6.9-0.61c-0.075-3.1-0.39-5.2-0.96-6.3-0.56-1.1-1.4-1.7-2.4-1.7z" fill="#fff"/>
 <path d="m560 12c0.98-0.049 1.8-0.018 2.5 0.094 0.72 0.11 1.3 0.26 1.7 0.45l-0.34-6.7-2.7-0.8-0.1-2 9.7-1.9 0.94 0.086 1.6 31 2.8 1.2 0.098 2-9.6 1.6-0.76-0.051-0.13-2.5-0.22 0.011c-0.68 0.75-1.5 1.4-2.5 2-1 0.59-2.3 0.92-3.8 0.99-2.6 0.13-4.9-0.78-6.7-2.7-1.8-2-2.8-5-3-9.1-0.22-4.3 0.65-7.6 2.6-10 2-2.3 4.6-3.6 8-3.7zm3 21c0.47-0.024 0.9-0.1 1.3-0.24 0.41-0.14 0.73-0.3 0.95-0.49l-0.87-17c-0.31-0.19-0.67-0.35-1.1-0.48s-0.94-0.17-1.6-0.14c-1.3 0.066-2.3 0.87-3 2.4-0.69 1.5-0.95 4.1-0.77 7.6 0.16 3.2 0.66 5.4 1.5 6.7 0.87 1.3 2.1 1.9 3.6 1.9z" fill="#fff"/>
 <path d="m520 36 2.5-1.1 0.071-17-2.7-1.1 8e-3 -1.9 9.7-1.4 0.98 0.093-0.012 2.8 0.27 0.0011c1-0.91 2.3-1.7 3.8-2.2 1.5-0.56 3-0.83 4.5-0.83 2.1 0.0085 3.6 0.53 4.6 1.6 0.97 1 1.5 2.7 1.4 4.9l-0.062 15 2.5 1.2-8e-3 2-13-0.053 8e-3 -2 2.3-1 0.058-14c5e-3 -1.2-0.26-2.1-0.79-2.6-0.5-0.54-1.3-0.81-2.3-0.81-0.65-0.0027-1.3 0.069-1.8 0.21-0.53 0.15-1 0.32-1.5 0.53l-0.069 17 2.2 1-8e-3 2-13-0.053 8e-3 -2z" fill="#fff"/>
 <path d="m518 25-14 2.5c0.54 2.4 1.5 4.2 2.7 5.2 1.3 1 3.2 1.3 5.9 0.83 1.3-0.22 2.3-0.53 3.3-0.93 0.92-0.43 1.8-0.94 2.5-1.5l0.38 2.2c-0.48 1-1.5 2.1-3 3.3-1.5 1.1-3.3 1.9-5.6 2.3-4.1 0.71-7.3 0.1-9.8-1.8-2.5-1.9-4.1-4.9-4.7-8.8-0.67-3.8-0.13-7 1.6-9.6 1.8-2.6 4.5-4.3 8.1-4.9 3.4-0.6 6.2-0.091 8.3 1.5 2.1 1.6 3.6 4.7 4.4 9.2l0.11 0.61zm-13-8.9c-1.1 0.19-1.9 1-2.3 2.4-0.41 1.4-0.36 3.6 0.15 6.6l6.7-1.6c-0.53-3-1.2-5.1-1.9-6.1-0.72-1-1.6-1.5-2.7-1.3z" fill="#fff"/>
 <path d="m494 26c-0.074 2.4-0.45 4.4-1.1 6.1-0.65 1.7-1.5 3.1-2.6 4.1-1 1-2.3 1.8-3.7 2.2-1.4 0.46-2.8 0.67-4.3 0.62-0.95-0.03-1.8-0.14-2.5-0.35-0.74-0.17-1.3-0.37-1.6-0.58l-0.21 6.7 2.9 1.2-0.062 2-14-0.42 0.062-2 2.6-0.99 0.83-27-2.6-1.4 0.058-1.9 9.8-1.1 0.98 0.12-0.088 2.8 0.27 0.0083c0.64-0.72 1.6-1.4 2.7-2 1.2-0.62 2.6-0.9 4.2-0.85 1.2 0.036 2.2 0.29 3.2 0.77 1 0.45 1.9 1.2 2.6 2.1 0.74 0.97 1.3 2.2 1.7 3.8 0.4 1.6 0.56 3.4 0.49 5.6zm-8.5 0.62c0.1-3.3-0.21-5.6-0.94-6.8-0.73-1.2-1.8-1.9-3.1-1.9-0.5-0.016-0.98 0.044-1.4 0.18-0.45 0.1-0.84 0.24-1.2 0.41l-0.53 17c0.29 0.22 0.62 0.42 1 0.61 0.38 0.16 0.85 0.25 1.4 0.27 1.4 0.043 2.5-0.66 3.4-2.1 0.85-1.5 1.3-4 1.4-7.6z" fill="#fff"/>
 <path d="m466 28-14 0.36c0.18 2.5 0.83 4.3 2 5.5s3 1.7 5.7 1.7c1.3-0.031 2.4-0.18 3.4-0.44 0.97-0.29 1.9-0.67 2.7-1.1l0.054 2.2c-0.63 0.93-1.8 1.9-3.4 2.8-1.6 0.9-3.6 1.4-5.9 1.4-4.1 0.1-7.3-0.98-9.4-3.2-2.2-2.3-3.3-5.4-3.4-9.4-0.095-3.9 0.91-7 3-9.3 2.1-2.3 5.1-3.5 8.8-3.6 3.5-0.086 6.2 0.83 8 2.7 1.9 1.9 2.8 5.2 3 9.8l0.015 0.62zm-11-11c-1.1 0.028-2 0.72-2.7 2.1-0.62 1.3-0.89 3.5-0.82 6.5l6.9-0.61c-0.076-3.1-0.4-5.2-0.96-6.3-0.56-1.1-1.4-1.7-2.4-1.7z" fill="#fff"/>
 <path d="m427 15c0.98-0.024 1.8 0.029 2.5 0.16 0.71 0.13 1.3 0.29 1.7 0.49l-0.16-6.7-2.7-0.87-0.05-2 9.8-1.7 0.94 0.11 0.77 31 2.7 1.2 0.049 2-9.7 1.3-0.76-0.07-0.062-2.5-0.22 0.0055c-0.69 0.73-1.6 1.4-2.6 2-1 0.56-2.3 0.86-3.8 0.89-2.6 0.065-4.9-0.9-6.7-2.9-1.8-2-2.7-5.1-2.8-9.2-0.11-4.3 0.85-7.6 2.9-9.9 2-2.3 4.7-3.5 8.1-3.5zm2.4 21c0.47-0.012 0.9-0.082 1.3-0.21 0.41-0.13 0.74-0.28 0.97-0.47l-0.42-17c-0.3-0.2-0.66-0.37-1.1-0.51s-0.94-0.2-1.6-0.18c-1.3 0.032-2.3 0.81-3.1 2.3-0.73 1.5-1.1 4.1-0.97 7.6 0.078 3.2 0.52 5.4 1.3 6.8 0.83 1.3 2 2 3.6 2z" fill="#fff"/>
 <path d="m386 38 2.6-0.93 1.5-17-2.6-1.3 0.16-1.9 9.8-0.56 0.97 0.17-0.25 2.8 0.27 0.024c1.1-0.82 2.4-1.5 4-1.9 1.5-0.43 3-0.58 4.5-0.45 2.1 0.19 3.6 0.84 4.5 2 0.88 1.1 1.2 2.8 1 5l-1.4 15 2.4 1.4-0.17 1.9-13-1.1 0.17-1.9 2.3-0.82 1.2-14c0.11-1.2-0.081-2.1-0.57-2.7-0.45-0.58-1.2-0.91-2.2-1-0.65-0.058-1.3-0.037-1.8 0.061-0.54 0.1-1.1 0.23-1.6 0.4l-1.5 17 2.1 1.2-0.17 1.9-13-1.1 0.17-1.9z" fill="#fff"/>
 <path d="m378 6.6c1.2-0.028 2.2 0.35 3.1 1.1 0.88 0.78 1.3 1.8 1.4 2.9 0.03 1.2-0.38 2.2-1.2 3-0.84 0.79-1.8 1.2-3 1.2-1.2 0.029-2.2-0.33-3-1.1-0.85-0.78-1.3-1.8-1.3-2.9-0.029-1.2 0.36-2.2 1.2-3s1.8-1.2 3-1.3zm-6.3 12 10-1.7 0.85 0.068 0.53 22 2.5 1 0.047 1.9-13 0.32-0.047-1.9 2.5-1.2-0.43-17-2.7-1-0.047-1.9z" fill="#fff"/>
 <path d="m353 43c0.19-1.9 0.36-3.9 0.52-6 0.19-2 0.31-4.1 0.38-6.2l8-0.084c-0.33 2.1-1.1 4.2-2.2 6.3-1.1 2.1-2.4 4.1-3.9 5.9l-2.8 0.029z" fill="#fff"/>
 <path d="m343 23c2.6 0.6 4.4 1.5 5.6 2.7s1.8 3 1.8 5.3c0.028 2.7-0.79 4.8-2.5 6.3-1.7 1.5-4.3 2.3-7.9 2.3-1.4 0.015-2.9-0.088-4.4-0.31-1.5-0.19-2.8-0.5-4.1-0.94l-0.33-6.4 2.9-0.031 1.9 4.2c0.48 0.26 1 0.46 1.7 0.6 0.65 0.14 1.4 0.21 2.1 0.2 1.4-0.014 2.4-0.31 3-0.88 0.65-0.57 0.96-1.3 0.95-2.2-8e-3 -0.8-0.27-1.4-0.78-1.9-0.48-0.47-1.3-0.85-2.6-1.1l-1.6-0.38c-2.6-0.62-4.5-1.6-5.8-2.9-1.3-1.3-1.9-3.1-1.9-5.4-0.025-2.4 0.77-4.3 2.4-5.8 1.6-1.6 4.2-2.4 7.6-2.4 1.4-0.015 2.8 0.06 4.1 0.22 1.3 0.13 2.5 0.37 3.4 0.72l0.062 6-2.9 0.031-1.3-3.8c-0.36-0.17-0.8-0.3-1.3-0.39-0.5-0.084-1.1-0.12-1.7-0.12-1.2 0.012-2.1 0.27-2.7 0.78-0.59 0.51-0.88 1.2-0.87 2.1 8e-3 0.8 0.27 1.4 0.78 1.9 0.51 0.44 1.4 0.83 2.8 1.2l1.6 0.38z" fill="#fff"/>
 <path d="m322 24c2.6 0.6 4.4 1.5 5.6 2.7 1.2 1.2 1.8 3 1.8 5.3 0.029 2.7-0.79 4.8-2.5 6.3-1.7 1.5-4.3 2.3-7.9 2.3-1.4 0.015-2.9-0.088-4.4-0.31-1.5-0.19-2.8-0.5-4.1-0.94l-0.33-6.4 2.9-0.031 1.9 4.2c0.48 0.26 1 0.46 1.7 0.6 0.65 0.14 1.4 0.21 2.1 0.2 1.4-0.014 2.4-0.31 3-0.88 0.65-0.57 0.96-1.3 0.96-2.2-9e-3 -0.8-0.27-1.4-0.78-1.9-0.48-0.47-1.3-0.85-2.6-1.1l-1.6-0.38c-2.6-0.62-4.5-1.6-5.8-2.9-1.3-1.3-1.9-3.1-1.9-5.4-0.025-2.4 0.77-4.3 2.4-5.8 1.6-1.6 4.2-2.4 7.6-2.4 1.4-0.015 2.8 0.06 4.1 0.22 1.3 0.13 2.5 0.37 3.4 0.72l0.062 6-2.9 0.031-1.3-3.8c-0.36-0.17-0.8-0.3-1.3-0.39-0.5-0.084-1.1-0.12-1.7-0.12-1.2 0.012-2.1 0.27-2.7 0.78-0.59 0.51-0.88 1.2-0.87 2.1 8e-3 0.8 0.27 1.4 0.78 1.9 0.51 0.44 1.4 0.83 2.8 1.2l1.6 0.38z" fill="#fff"/>
 <path d="m308 28-14 0.15c0.14 2.5 0.77 4.3 1.9 5.5s3 1.8 5.7 1.8c1.3-0.013 2.4-0.14 3.4-0.39 0.98-0.28 1.9-0.64 2.7-1.1l0.023 2.2c-0.64 0.93-1.8 1.8-3.4 2.7-1.6 0.88-3.6 1.3-5.9 1.4-4.1 0.043-7.2-1.1-9.4-3.4-2.1-2.3-3.2-5.5-3.3-9.4-0.041-3.9 1-7 3.1-9.2 2.2-2.3 5.1-3.5 8.8-3.5 3.5-0.037 6.2 0.91 8 2.8 1.8 1.9 2.8 5.2 2.8 9.8l6e-3 0.62zm-11-11c-1.1 0.012-2 0.69-2.7 2-0.64 1.3-0.94 3.5-0.91 6.5l6.9-0.52c-0.032-3.1-0.32-5.2-0.87-6.3-0.54-1.2-1.4-1.7-2.4-1.7z" fill="#fff"/>
 <path d="m270 6.7 9.8-2 0.98 0.079 1.8 32 2.6 1 0.11 2-13 0.73-0.11-2 2.5-1.3-1.5-28-2.8-0.87-0.11-2z" fill="#fff"/>
 <path d="m253 16 9.8-0.33 0.79 0.18-0.62 5.5 0.22 0.025c0.87-1.9 1.9-3.3 3-4.1 1.1-0.86 2.4-1.2 3.8-1.1 0.21 0.023 0.42 0.063 0.66 0.12 0.23 0.056 0.4 0.12 0.52 0.19l-0.82 7.3c-0.2-0.082-0.49-0.16-0.87-0.23-0.35-0.069-0.72-0.13-1.1-0.17-1.1-0.12-2.1-0.17-2.9-0.15s-1.7 0.14-2.4 0.36l-1.6 14 3.1 1.5-0.21 1.9-14-1.6 0.21-1.9 2.6-0.82 1.9-17-2.5-1.4 0.21-1.9z" fill="#fff"/>
 <path d="m235 27 5.3-1.1-0.035-3.3c-0.017-1.7-0.33-2.9-0.93-3.6-0.6-0.73-1.6-1.1-3-1.1-0.41 0.0044-0.8 0.038-1.2 0.1-0.33 0.033-0.65 0.096-0.98 0.19l-1.6 5.4-3.6 0.037-0.07-6.7c1.2-0.37 2.5-0.72 3.9-1.1 1.5-0.37 3.2-0.57 5.1-0.59 3.3-0.034 5.8 0.47 7.6 1.5 1.8 1 2.8 2.9 2.8 5.7l0.15 14 2.3 1.1 0.017 1.6c-0.41 0.3-1 0.56-1.9 0.78-0.83 0.25-1.7 0.37-2.7 0.38-1.5 0.016-2.7-0.22-3.5-0.72-0.81-0.52-1.4-1.2-1.7-2.1l-0.27 0.0028c-0.7 0.93-1.6 1.7-2.6 2.2-1 0.51-2.3 0.78-3.9 0.8-2 0.022-3.7-0.54-5-1.7-1.3-1.2-1.9-2.8-2-4.9-0.022-2 0.6-3.7 1.9-4.8 1.3-1.2 3.2-2 5.8-2.5zm3.3 9.7c0.47-5e-3 0.89-0.069 1.2-0.19 0.38-0.15 0.68-0.33 0.88-0.54l-0.079-7.6-2 0.24c-1.1 0.13-1.9 0.52-2.4 1.2-0.5 0.66-0.74 1.7-0.72 3 0.016 1.5 0.31 2.6 0.88 3.1 0.57 0.5 1.3 0.74 2.2 0.73z" fill="#fff"/>
 <path d="m225 29-14 0.15c0.14 2.5 0.77 4.3 1.9 5.5s3 1.8 5.7 1.8c1.3-0.013 2.4-0.14 3.4-0.39 0.98-0.28 1.9-0.64 2.7-1.1l0.023 2.2c-0.64 0.93-1.8 1.8-3.4 2.7-1.6 0.88-3.6 1.3-5.9 1.4-4.1 0.043-7.2-1.1-9.4-3.4-2.1-2.3-3.2-5.5-3.3-9.4-0.041-3.9 1-7 3.1-9.2 2.2-2.3 5.1-3.5 8.8-3.5 3.5-0.037 6.2 0.91 8 2.8s2.8 5.2 2.8 9.8l7e-3 0.62zm-11-11c-1.1 0.012-2 0.69-2.7 2-0.64 1.3-0.94 3.5-0.91 6.5l6.9-0.52c-0.032-3.1-0.32-5.2-0.87-6.3-0.55-1.2-1.4-1.7-2.4-1.7z" fill="#fff"/>
 <path d="m197 38 3.2 0.99 0.1 1.9-14 0.74-0.1-1.9 2.5-1.3-0.95-18-2.7 0.14-0.11-2 2.6-1.4-0.13-2.5c-0.08-1.5 0.1-2.8 0.55-3.9 0.48-1.1 1.1-2 1.9-2.7 0.85-0.73 1.8-1.3 3-1.6 1.2-0.36 2.4-0.57 3.7-0.64 1-0.055 2-0.076 2.9-0.064 0.89 0.012 1.9 0.09 3.1 0.23l0.35 6.6-3 0.16-1.3-4.3c-0.21-0.078-0.42-0.13-0.63-0.14-0.18-0.02-0.37-0.025-0.58-0.014-0.77 0.041-1.3 0.34-1.7 0.89-0.33 0.52-0.46 1.5-0.38 2.8l0.25 4.7 4.5-0.24 0.18 3.3-4.5 0.24 0.96 18z" fill="#fff"/>
 <path d="m159 19-3.5-0.051 0.031-2.1 3.5-1.1 0.065-4.4 8.1-1.2-0.084 5.6 5.4 0.079-0.048 3.3-5.4-0.079-0.21 14c-0.018 1.2 0.19 2.2 0.63 2.7 0.47 0.54 1.4 0.82 2.7 0.84 0.38 0.0057 0.78-0.0032 1.2-0.027 0.44-0.023 0.8-0.048 1.1-0.073l-0.034 2.3c-0.72 0.4-1.7 0.75-2.8 1-1.1 0.31-2.4 0.45-3.7 0.43-2.4-0.035-4.2-0.64-5.4-1.8-1.2-1.2-1.7-3.1-1.7-5.7l0.21-14z" fill="#fff"/>
 <path d="m136 17 9.6-2.2 0.8 0.024 0.44 5.5 0.22-0.018c0.49-2 1.2-3.6 2.1-4.6 0.96-1.1 2.1-1.6 3.5-1.8 0.21-0.017 0.43-0.02 0.67-0.0091 0.24 0.011 0.42 0.041 0.54 0.09l0.59 7.3c-0.21-0.042-0.51-0.063-0.9-0.062-0.36-1e-3 -0.73 0.014-1.1 0.045-1.1 0.088-2.1 0.23-2.9 0.41-0.85 0.19-1.6 0.46-2.3 0.81l1.1 14 3.4 0.84 0.15 1.9-14 1.1-0.15-1.9 2.4-1.3-1.4-17-2.7-0.85-0.15-1.9z" fill="#fff"/>
 <path d="m121 40c-1.8-0.24-3.5-0.75-4.9-1.5-1.4-0.82-2.6-1.8-3.6-3.1-0.94-1.3-1.6-2.7-2-4.4-0.38-1.7-0.44-3.5-0.19-5.4 0.25-1.9 0.77-3.7 1.6-5.1 0.79-1.5 1.8-2.7 3-3.6 1.2-0.95 2.6-1.6 4.2-2 1.6-0.39 3.3-0.46 5.2-0.23 1.9 0.24 3.5 0.75 5 1.5 1.5 0.79 2.7 1.8 3.6 3 0.95 1.2 1.6 2.6 2 4.3 0.42 1.6 0.5 3.4 0.25 5.4-0.25 1.9-0.77 3.7-1.6 5.2-0.79 1.5-1.8 2.8-3 3.7-1.2 0.95-2.6 1.6-4.3 2-1.6 0.39-3.3 0.46-5.2 0.23zm3-23c-0.59-0.076-1.1 0.0023-1.6 0.24-0.48 0.24-0.93 0.72-1.4 1.4-0.42 0.72-0.81 1.7-1.2 3-0.32 1.3-0.6 2.9-0.86 4.9-0.26 2-0.41 3.7-0.43 5.1 5e-3 1.3 0.12 2.4 0.34 3.2 0.22 0.81 0.55 1.4 0.98 1.8 0.43 0.35 0.94 0.57 1.5 0.65 0.59 0.076 1.1-0.0042 1.6-0.24 0.51-0.26 0.96-0.76 1.4-1.5 0.43-0.75 0.8-1.8 1.1-3.1 0.35-1.3 0.65-3 0.92-5 0.26-2 0.38-3.7 0.38-5 0.024-1.3-0.078-2.4-0.31-3.2-0.2-0.8-0.51-1.4-0.94-1.7-0.4-0.35-0.91-0.57-1.5-0.65z" fill="#fff"/>
 <path d="m109 27c-0.18 2.4-0.64 4.4-1.4 6-0.72 1.7-1.6 3-2.8 4-1.1 0.99-2.3 1.7-3.8 2.1-1.4 0.4-2.8 0.54-4.3 0.43-0.95-0.071-1.8-0.22-2.5-0.46-0.73-0.2-1.3-0.42-1.6-0.66l-0.5 6.6 2.8 1.3-0.15 2-14-1 0.15-2 2.6-0.87 2-27-2.5-1.5 0.14-1.9 9.8-0.69 0.97 0.16-0.21 2.8 0.27 0.02c0.68-0.69 1.6-1.3 2.8-1.8 1.2-0.56 2.6-0.78 4.2-0.66 1.2 0.087 2.2 0.39 3.2 0.91 1 0.49 1.8 1.2 2.5 2.2 0.7 1 1.2 2.3 1.5 3.9 0.33 1.6 0.41 3.4 0.24 5.6zm-8.5 0.25c0.25-3.3 0.035-5.6-0.64-6.8-0.68-1.3-1.7-2-3.1-2.1-0.5-0.038-0.98 3e-4 -1.4 0.11-0.45 0.085-0.85 0.2-1.2 0.36l-1.3 17c0.28 0.23 0.61 0.45 0.98 0.65 0.37 0.18 0.84 0.29 1.4 0.33 1.4 0.1 2.5-0.55 3.4-2 0.91-1.4 1.5-3.9 1.8-7.5z" fill="#fff"/>
 <path d="m81 27c-0.29 2.4-0.85 4.3-1.7 6-0.8 1.6-1.8 2.9-2.9 3.9-1.1 0.94-2.4 1.6-3.9 1.9-1.4 0.33-2.9 0.41-4.3 0.23-0.94-0.12-1.8-0.31-2.5-0.57-0.72-0.24-1.2-0.48-1.6-0.73l-0.81 6.6 2.7 1.4-0.24 2-13-1.6 0.24-2 2.6-0.75 3.2-26-2.4-1.6 0.23-1.9 9.8-0.23 0.96 0.21-0.34 2.8 0.26 0.032c0.71-0.66 1.7-1.2 2.9-1.7 1.2-0.51 2.6-0.66 4.3-0.46 1.1 0.14 2.2 0.49 3.1 1.1 0.98 0.54 1.8 1.3 2.4 2.4 0.65 1 1.1 2.3 1.3 3.9 0.25 1.6 0.25 3.5-0.019 5.6zm-8.5-0.15c0.4-3.3 0.29-5.6-0.33-6.8-0.62-1.3-1.6-2-3-2.2-0.5-0.061-0.98-0.045-1.4 0.048-0.46 0.064-0.86 0.16-1.2 0.3l-2.1 17c0.27 0.24 0.58 0.47 0.94 0.7 0.36 0.19 0.83 0.33 1.4 0.39 1.4 0.17 2.6-0.43 3.5-1.8 0.97-1.4 1.7-3.9 2.1-7.4z" fill="#fff"/>
 <path d="m28 19-2.7-1.2v-2l9.7-1.3 1.1 0.089v18c0 1.2 0.27 2 0.8 2.4 0.53 0.41 1.3 0.62 2.2 0.62 0.59 0 1.1-0.059 1.6-0.18 0.53-0.15 1-0.33 1.5-0.53v-16l-2.7-1.2v-2l9.7-1.3 0.98 0.089v21l2.6 1.3v1.8l-9.6 1.1-0.98-0.089v-2.9h-0.27c-0.89 0.83-2 1.6-3.2 2.2-1.2 0.59-2.6 0.89-4.1 0.89-2.3 0-4-0.64-5.2-1.9-1.1-1.3-1.6-2.9-1.6-5v-14z" fill="#fff"/>
 <path d="m11 41c-1.8 0-3.6-0.15-5.4-0.44-1.8-0.27-3.2-0.65-4.4-1.2l-0.44-8.1h3.8l1.7 5.9c0.56 0.27 1.2 0.47 1.9 0.62 0.74 0.12 1.5 0.18 2.4 0.18 1.9 0 3.2-0.37 4.1-1.1 0.89-0.74 1.3-1.8 1.3-3.2 0-1.3-0.46-2.3-1.4-3-0.92-0.74-2.4-1.5-4.3-2.3l-1.7-0.71c-2.5-1-4.4-2.3-5.7-3.8-1.3-1.5-1.9-3.6-1.9-6.1 0-1.4 0.22-2.7 0.67-3.9 0.44-1.2 1.1-2.3 2.1-3.2 0.95-0.89 2.2-1.6 3.6-2.1 1.5-0.5 3.3-0.76 5.4-0.76 2.2 0 4 0.15 5.3 0.44 1.4 0.27 2.5 0.55 3.3 0.84l0.44 7.8h-3.6l-1.6-5.6c-0.41-0.18-0.95-0.31-1.6-0.4-0.65-0.12-1.4-0.18-2.3-0.18-1.5 0-2.7 0.37-3.4 1.1-0.77 0.71-1.2 1.7-1.2 2.9 0 1.1 0.37 2 1.1 2.7 0.77 0.68 2 1.3 3.8 2l1.8 0.71c1.4 0.53 2.6 1.1 3.7 1.6 1.1 0.53 2.1 1.2 2.8 2 0.77 0.74 1.3 1.6 1.7 2.7 0.41 1.1 0.62 2.4 0.62 3.9 0 3.4-1.1 6-3.4 7.9-2.3 1.8-5.4 2.8-9.3 2.8z" fill="#fff"/>
</svg>
```

Reducing the precision to 2 decimal places brings the filesize down from 36k to 20k ... I've also added a role attribute to the top level svg element to ensure the contents of the svg aren't read out by a screen reader as well as adding a title element for screen readers to read out